### DISCUSSION TITLE
What will Web support look like?

### DISCUSSION THREAD

**Discussion by andria-dev** [zed-industries/zed#6462] [Feature Requests]
I recently saw on [Zed's roadmap](https://zed.dev/roadmap) that the Adoption team will be focused on a number of things including Linux and Web support—I was a bit surprised that Windows wasn't on there, but it is what it is—and I'm wondering what Web support will be. Will it be like any of the following, or will it be something different?

- [VSCode for Web](vscode.dev)—an editor that saves to your filesystem or pushes to GitHub and lacks a terminal.
- [CodeMirror](https://codemirror.net/)—a configurable embeddable/scriptable editor.
- [GitHub Codespaces](https://github.com/codespaces) or [Glitch](https://glitch.com/)—Cloud environment with an integrated terminal.

Also, will Zed for Web be hosted by y'all?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Reply by andria-dev** [reply]
this was from over 2 years ago...

**Reply by andria-dev** [reply]
Like I don't really care that much, but you had 2 years to answer. I don't see how Discord helps at all in this scenario.
